src/api/users.ts

```diff
@@ -0,0 +1,9 @@
+import { api } from ".";
+import { User } from "../types";
+import { CreateUserDto } from "../types/dto";
```

Comment:
I would rename CreateUserDTO to UserRegisterRequestDTO

Reply:
done